src/RoutesWrap.tsx

```diff
@@ -39,7 +39,7 @@ export const CommonRoutesFeatures = ({
 
   useEffect(() => {
     initLanguageWatcher();
-    return forceRun(urlStoreAtom);
+    // return forceRun(urlStoreAtom);
```

Comment:
what is the reason for commenting instead of removing? you can specify it in code comments